mypy.ini

```diff
@@ -167,7 +167,17 @@ ignore_errors=True
 [mypy-pandas.io.stata]
 ignore_errors=True
 
+<<<<<<< HEAD
+[mypy-pandas.tseries.frequencies]
+ignore_errors=True
+
+[mypy-pandas.tseries.holiday]
+ignore_errors=True
+
+[mypy-pandas.tseries.offsets]
+=======
 [mypy-pandas.plotting._core]
+>>>>>>> origin/master
```

Comment:
Merge conflict vestiges?

Reply:
Oops. Changed IDEs, forgot to turn on autosave.